# Extract the entropy map for a specific class instance
# author: Tim Patten, t.patten@acfr.usyd.edu.au

# the class name
std_msgs/String class_type

# the instance name
std_msgs/String instance_name

---

# model point cloud, octomap octree and entropy map for specific class and instance
sensor_msgs/PointCloud2 cloud
std_msgs/String octree_file
ViewpointEntropy[] entropy_map
